run.sh

```diff
@@ -3,26 +3,25 @@
 rm -rf work/*
 
 luser=${LTI_LINKY_USER?"No Linky username specified"}
-lpass=${LTI_LINKY_PASS?"No Linky password specified"}
```

Comment:
By commenting this line, the script won't check if a password is specified anymore.
Could you please reactivate the check so that user can be explicitly notified that something's missing ?

Reply:
Yes we can undone the drop line if you want.